glommio/src/nop.rs

```diff
@@ -0,0 +1,31 @@
+//! Provides a [`NopSubmitter`] which can be used to submit nop operations to the ring. This is mainly useful for benchmarking Glommio.
```

Comment:
Do we need a license header for these?

Reply:
We don't have license headers is any of our files. There is the datadog snippet but that's not a license. I don't expect non-DD people to add that - although you can, for consistency, if you want.